dispatcher/dispatcher_test.go

```diff
@@ -16,26 +16,29 @@ import (
 	"github.com/stretchr/testify/assert"
 	"google.golang.org/protobuf/proto"
 
-	"github.com/iotexproject/iotex-core/config"
 	"github.com/iotexproject/iotex-proto/golang/iotexrpc"
 	"github.com/iotexproject/iotex-proto/golang/iotextypes"
 	"github.com/iotexproject/iotex-proto/golang/testingpb"
 )
 
+var (
+	defaultChainID uint32 = 1
```

Comment:
this should be defined in chain.Default (when moving struct Chain {} out of config)

Reply:
also, let's move this part: https://github.com/iotexproject/iotex-core/blob/master/config/config.go#L77-L88
into p2p, define a p2p.Default for it

Reply:
and also move https://github.com/iotexproject/iotex-core/blob/master/config/config.go#L77-L88
into p2p, define a p2p.Default for it

Reply:
> his should be defined in chain.Default (when moving struct Chain {} out of config

Should config.Default move out of config.go?
In #2695, the network default config is kept in config.go

Reply:
config.Default stays in config.go, just become like 
```
Default = Config{
    Network: p2p.Default,
    Dispatcher: dispatcher.Default
}
```